receiver/googlecloudspannerreceiver/internal/metadataconfig/metadata.yaml

```diff
@@ -446,4 +500,22 @@ metadata:
         value_type: "float"
         data:
           type: "gauge"
-        unit: "byte"
\ No newline at end of file
+        unit: "byte"
+      - name: "ninety_nine_percentile_total_latency"
+        column_name: "NINETY_NINE_PERCENTILE_TOTAL_LATENCY"
+        value_type: "null_float"
+        data:
+          type: "gauge"
+        unit: "second"
+      - name: "ninety_five_percentile_total_latency"
+        column_name: "NINETY_FIVE_PERCENTILE_TOTAL_LATENCY"
+        value_type: "null_float"
+        data:
+          type: "gauge"
+        unit: "second"
+      - name: "fifty_percentile_total_latency"
+        column_name: "FIFTY_PERCENTILE_TOTAL_LATENCY"
+        value_type: "null_float"
+        data:
+          type: "gauge"
+        unit: "second"
```

Comment:
Nitpick: it would be good to end the file with a newline.

Reply:
Done